src/core/function_messages_cqrs.tf

```diff
@@ -106,13 +109,14 @@ module "function_messages_cqrs" {
   app_settings = merge(
     local.function_messages_cqrs.app_settings, {
       // disable listeners on staging slot
-      "AzureWebJobs.CosmosApiMessageStatusChangeFeedForView.Disabled" = "0"
-      "AzureWebJobs.HandleMessageViewUpdateFailures.Disabled"         = "0"
-      "AzureWebJobs.UpdateCosmosMessageView.Disabled"                 = "0"
-      "AzureWebJobs.UpdatePaymentOnMessageView.Disabled"              = "0"
-      "AzureWebJobs.HandlePaymentUpdateFailures.Disabled"             = "0"
-      "AzureWebJobs.CosmosApiMessagesChangeFeed.Disabled"             = "0"
-      "AzureWebJobs.HandleMessageChangeFeedPublishFailures.Disabled"  = "0"
+      "AzureWebJobs.CosmosApiMessageStatusChangeFeedForView.Disabled"     = "0"
+      "AzureWebJobs.CosmosApiMessageStatusChangeFeedForReminder.Disabled" = "1"
```

Comment:
Do we want to add it as Disabled?

Reply:
yep because reminder is not ready to receive events :)